Content.Shared/CombatMode/DisarmedEvent.cs

```diff
@@ -16,5 +16,20 @@ public sealed class DisarmedEvent : HandledEntityEventArgs
         ///     Probability for push/knockdown.
         /// </summary>
         public float PushProbability { get; init; }
+
+        /// <summary>
+        ///     Whether the entity was successfully disarmed.
+        /// </summary>
+        public bool IsDisarmed { get; set; }
+
+        /// <summary>
+        ///     Whether the entity was successfully shoved.
+        /// </summary>
+        public bool IsShoved { get; set; }
+
```

Comment:
Wouldn't these be better just being prefixes?

Reply:
felt potentially wrong passing though strings but it would be fine
is it fine to be passing stuff back though events like this?

Reply:
You're just adding a single bool for every different handler which is just going to bloat to infinity when all you want is a string prefix for the popup.